CMakeLists.txt

```diff
@@ -79,6 +79,8 @@ cmake_dependent_option(BUILD_SAMPLES "Build samples?" ON "BUILD_DISPATCHER" OFF
 cmake_dependent_option(BUILD_TOOLS "Build tools?" "${BUILD_ALL}" "BUILD_SAMPLES" OFF)
 option(BUILD_TESTS "Build tests?" "${BUILD_ALL}")
 
+option(RUN_TESTS "Run tests during build?" OFF )
```

Comment:
You need cmake_dependent_option since you can't run tests if your don't build them:
`cmake_dependent_option(RUN_TESTS "Run tests?" OFF "BUILD_TESTS" OFF)`